src/Views/Appearance.vala

```diff
@@ -38,6 +38,19 @@ public class Appearance : Gtk.Grid {
         row_spacing = 6;
         margin_start = margin_end = 6;
 
+        var dark_label = new Gtk.Label (_("Prefer dark style:"));
```

Comment:
I guess we probably shouldn't even construct this UI unless the DBus API exists. Or at least we shouldn't pack it into the grid

Reply:
@danrabbit I was wondering that and went back and forth. It seemed to be a little more confusing when the `attach` methods were missing up above, but I can move them and add a comment for clarity.